JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -164,7 +165,7 @@ private async Task UpdateMessageVisibilityTimeout(SQSMessage message, string rec
                 }
                 catch (Exception ex)
                 {
-                    _log.LogError(0, ex, $"Failed to update message visibility timeout by {visibilityTimeoutSeconds} seconds");
+                    _logger.LogError(0, ex, "Failed to update message visibility timeout by {visibilityTimeoutSeconds} seconds", visibilityTimeoutSeconds);
```

Comment:
I believe the general convention is to use PascalCase for the names of the properties.